bots/bot.go

```diff
@@ -0,0 +1,36 @@
+package bot
+
+import "crypto/tls"
+
+type BotHandleFunc func(string, []byte)
+
+type Bot interface {
+	HandleRegistration(string, []byte) string
+	Run()
+	Stop()
+}
+
+func SendRequestToCONIKS(address string, msg []byte) ([]byte, error) {
+	conf := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+
+	conn, err := tls.Dial("tcp", address, conf)
+	if err != nil {
+		return nil, err
+	}
+	defer conn.Close()
+
+	_, err = conn.Write(msg)
```

Comment:
The first pointer only writes 2 chars and the 2nd method returns the number of written bytes to the callee. Oh, and actually, they ignore it their, too. But this doesn't really matter: we need to make sure we `Write` (and probably `Read`) what we want to send (and recv.). I skimmed through the tls code: https://golang.org/src/crypto/tls/conn.go#L718 Looks like it's OK to ignore the number of bytes written... Sorry for the noise.